src/components/HistoryPagination.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <s-pagination
+    class="history-pagination"
+    layout="slot"
+    :current-page.sync="currentPage"
+    :page-size="pageAmount"
+    :total-text="totalText"
+  >
+    <span class="el-pagination__total">{{ totalText }}</span>
+    <s-button
+      type="link"
+      :tooltip="t('history.firstText')"
+      :disabled="isFirstPage"
+      @click="handlePaginationClick(PaginationButton.First)"
+    >
+      <s-icon name="chevrons-left-16" size="14" />
+    </s-button>
+    <s-button
+      type="link"
+      :tooltip="t('history.prevText')"
+      :disabled="isFirstPage"
+      @click="handlePaginationClick(PaginationButton.Prev)"
+    >
+      <s-icon name="chevron-left-16" size="14" />
+    </s-button>
+    <s-button
+      type="link"
+      :tooltip="t('history.nextText')"
+      :disabled="isLastPage"
+      @click="handlePaginationClick(PaginationButton.Next)"
+    >
+      <s-icon name="chevron-right-16" size="14" />
+    </s-button>
+    <s-button
+      type="link"
+      :tooltip="t('history.lastText')"
+      :disabled="isLastPage"
+      @click="handlePaginationClick(PaginationButton.Last)"
+    >
+      <s-icon name="chevrons-right-16" size="14" />
+    </s-button>
+  </s-pagination>
+</template>
+
+<script lang="ts">
+import { Mixins, Component, Prop } from 'vue-property-decorator';
+
+import TranslationMixin from './mixins/TranslationMixin';
+import { PaginationButton } from '../consts';
```

Comment:
may you put enum here (cuz it's used only here)?

Reply:
Sorry, but It is used in a couple of places, not only here.